packages/wallet-api-manifest-validator/tests/validator.test.ts

```diff
@@ -0,0 +1,267 @@
+        testManifest.params.networks[0].currency = "test";
+        expect(validateManifest(testManifest)).toBe(false);
+      });
+      test("params networks currency is correct", () => {
+        testManifest.params.networks[0].currency = "ethereum";
+        expect(validateManifest(testManifest)).toBe(true);
+      });
+    });
+
+    describe("chainID property", () => {
+      test("params networks chainID is incorrect", () => {
+        testManifest.params.networks[0].chainID = 1;
+        expect(validateManifest(testManifest)).toBe(false);
+      });
+      test("params networks chainID is correct", () => {
+        testManifest.params.networks[0].chainID = "1";
+        expect(validateManifest(testManifest)).toBe(true);
+      });
+    });
+
+    describe("nodeURL property", () => {
+      test("params networks nodeURL is incorrect", () => {
+        testManifest.params.networks[0].nodeURL = "test";
+        expect(validateManifest(testManifest)).toBe(false);
+      });
+      test("params networks nodeURL is correct", () => {
+        testManifest.params.networks[0].nodeURL = "test.test";
+        expect(validateManifest(testManifest)).toBe(true);
+      });
+    });
+  });
+});
+
+describe("plateform property", () => {
+  test("plateform is part of enum", () => {
+    testManifest.platform = "desktop";
+    expect(validateManifest(testManifest)).toBe(true);
+  });
+  test("plateform is not part of the enum", () => {
+    testManifest.platform = "test";
+    expect(validateManifest(testManifest)).toBe(false);
+  });
+});
+
+describe("plateform property", () => {
+  test("plateform is part of enum", () => {
+    testManifest.platform = "desktop";
+    expect(validateManifest(testManifest)).toBe(true);
+  });
+  test("plateform is not part of the enum", () => {
+    testManifest.platform = "test";
+    expect(validateManifest(testManifest)).toBe(false);
+  });
+});
+
+describe("categories property", () => {
+  test("categories is an array of enum", () => {
+    testManifest.categories = ["lend", "yield"];
+    expect(validateManifest(testManifest)).toBe(true);
+  });
+  test("categories is not array where values are not completly part of enum", () => {
+    testManifest.categories = ["lend", "test"];
+    expect(validateManifest(testManifest)).toBe(false);
+  });
+});
+
+describe("content property", () => {
+  test("content have i18n properties ", () => {
+    testManifest.content = {
+      shortDescription: {
+        en: "yes",
+        fr: "oui",
+      },
+      description: {
+        en: "yes",
+        fr: "oui",
+      },
+    };
+    expect(validateManifest(testManifest)).toBe(true);
+  });
+  test("content have wring i18n properties ", () => {
+    testManifest.content = {
+      shortDescription: {
+        EN: "yes",
+        a8: "oui",
+      },
+      description: {
+        e3: "yes",
+        Er: "oui",
+      },
+    };
+    expect(validateManifest(testManifest)).toBe(false);
+  });
+});
+*/
```

Comment:
If this is unused, it should be removed.

Regarding testing for this package, an efficient good first step could be to use test manifests (as started above with the 3 valid ones) to easily test the schema agains valid and invalid manifests.

We could latter add individual rules tests if need be, but it seems too cumbersome for now, especially since the schema might evolve drastically in the near / medium future.

```suggestion
```